CONTRIBUTING.md

```diff
@@ -108,7 +108,7 @@ executable will be in your current directory and can be run as shown in the
 
 + Run the test suite locally:
 
-  `go generate ./pkg/... && make check test testrace`
+  `make generate && make check test testrace`
```

Comment:
You didn't introduce this inaccuracy, but this is no longer reflective of the full test suite. The UI tests are invoked separately - currently I think the best way is `make -C pkg/ui` but perhaps we should add a new top-level target that does this for you.